fsharp-backend/tests/FuzzTests/ExecutePureFunctions.FuzzTests.fs

```diff
@@ -513,7 +517,8 @@ let equalsOCaml ((fn, args) : FnAndArgs) : bool =
         if not allowed2 then
           debugFn ()
 
-          print $"Got different error msgs:\n\"{aMsg}\"\n\nvs\n\"{eMsg}\"\n\n"
+          print
+            $"Got different error msgs:\n\"{aMsg}\"\n\n(F#) vs (OCaml)\n\"{eMsg}\"\n\n"
```

Comment:
it was confusing to me which error message was which - this helps, esp. as it matches the order in `allowedFuzzerErrors.jsonc`

Reply:
That's exactly why it's been confusing for me :)

Generally in "unit testing" literature (at least what I've read), `expected` comes first, then `actual`.

Reply:
Oh funny. Expecto puts `actual` first so that's what I've gotten used to now